sphinxcontrib/drawio.py

```diff
@@ -148,6 +148,11 @@ def render_drawio(self: SphinxTranslator, node: DrawIONode, in_filename: str,
     else:
         binary_path = "/opt/draw.io/drawio"
 
+    scale_args = ["--scale", scale]
```

Comment:
Could you please add this to the "unique_values" list?
This means the hash should be different when you use the same diagram twice with different scales - this way, they'll override one another.

[Note that I had plans to improve this so that we wouldn't have to change the hash for all the tests each time an option was added, but I never bothered fixing this - e.g. naming the exported image as `<originalfilename>-<page-index>=1-<etc>.<format>`. If you'd like to implement this so that tests aren't such a gigantic pain to update that's fine. If you have a better solution, feel free to do that too]

Reply:
´scale´ was already included in ´unique_values´. Or am I misunderstanding?

Reply:
Yup, sorry.